UI/styles.css

```diff
@@ -0,0 +1,534 @@
+:root {
+    --primary: #2196F3;
+    --secondary: #48BBFC;
+    --black: #3D3D3D;
+    --hover: #0475CE;
+    --white: #C4C4C4;
+}
+
+
+body {
+	background: #FAFAFA;
+	font-family: Lato, Roboto;
+}
+
+h1, .heading {
+	font-family: Work Sans;
+	font-size: 42px;
+	font-weight: bold;
+}
+
+nav {
+	background: #F4F4F4;
+}
+
+
+/* 48BBFC - blue
+   2196F3 - 
+   3D3D3D - black
+   #FAFAFA - 
+   #F4F4F4 - grey
+*/
+
+
+/* Margin Utilities */
+
+.mt1 {
+	margin-top: 0.25rem !important;
+}
+
+.mt2 {
+	margin-top: 0.5rem !important;
+}
+
+.mt3 {
+	margin-top: 1rem !important;
+}
+
+
+.mt4 {
+	margin-top: 1.5rem !important;
+}
+
+.mt5 {
+	margin-top: 3rem !important;
+}
+
+.mb1 {
+	margin-bottom: 0.25rem !important;
```

Comment:
why are we doing so many important tags please read this article https://css-tricks.com/when-using-important-is-the-right-choice/. Please remove them

Reply:
Thanks for this link but I knew that `!important` is good for utility classes, and this is what the .*mb1 and .mb2* are, they are utility classes to apply a margin of a particular magnitude to any element. Also the post confirmed this.

![image](https://user-images.githubusercontent.com/8234355/40111415-aeecddee-58fa-11e8-909b-303504a784ea.png)